services/utils/knack.py

```diff
@@ -1,6 +1,27 @@
 import arrow
 
 
+def get_metadata(client, app_id):
```

Comment:
this helper used to live in `utils.Postgrest`, which I've obliterated. this seems like a reasonable home.